### ISSUE TITLE
Add Algolia search

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/open.ribose.com#15]
For @strogonoff 

- Copy the following CSS/JS snippets and add them to your page 

```html
<!-- at the end of the HEAD --> 
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" /> 

<!-- at the end of the BODY --> 
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({ 
apiKey: '31ca4bad475130bdaee582a0b9f40806', 
indexName: 'metanorma', 
inputSelector: '### REPLACE ME ####', 
debug: false // Set debug to true if you want to inspect the dropdown 
}); 
</script> 
```

* Add a search input in your page if you don't have any yet. Then update the inputSelector value in JS snippet to a CSS selector that targets your search input field. 
* Optionally customize the look and feel by following the DocSearch documentation (https://community.algolia.com/docsearch/documentation/) 
* You can also check your configuration in our github repo (https://github.com/algolia/docsearch-configs/blob/master/configs/metanorma.json).

**Comment by strogonoff** [comment]
@ronaldtse Do you have Algolia API key and index name for Ribose Open hub site? Your sample code above appears to be for Metanorma project site (which is good and works, but I’d like to test the search feature on a hub site too before releasing it as part of OP framework).

**Comment by ronaldtse** [comment]
The API key is: 40a5c3929d2f6d51c9a062d2bdf00c20

The index however hasn't been created yet.

**Comment by ronaldtse** [comment]
@strogonoff could we get this done soon? Thanks!

**Comment by strogonoff** [comment]
@ronaldtse this is blocked because we need API key *and* a dedicated index name for each site. (Currently I don’t have Algolia index name for open.ribose.com 🤷‍♂️)

**Comment by ronaldtse** [comment]
Index name: prod_riboseopen

**Comment by strogonoff** [comment]
OK that was quick!

**Comment by strogonoff** [comment]
I get “Invalid Application-ID or API key” when I try searching while running the Jekyll site locally; pushing an update to see how it works in production. Should be live in a few min. Config used:

```
algolia_search:
  api_key: '40a5c3929d2f6d51c9a062d2bdf00c20'
  index_name: 'prod_riboseopen'
```

**Comment by strogonoff** [comment]
Yep, Algolia error on production site, either index name or API key is incorrect 🤷‍♂️ @ronaldtse could you check?

  api_key: '40a5c3929d2f6d51c9a062d2bdf00c20'
  index_name: 'prod_riboseopen'
